content/documentation/validations/index.adoc

```diff
@@ -687,10 +687,156 @@ include::/data/files/validation_examples/601.yaml[]
+https://istio.io/docs/reference/config/networking/sidecar/#IstioEgressListener[Sidecar EgressListener documentation, window="_blank"] +
+
+'''
+
+=== KIA1004 - This host has no matching entry in the service registry
+
+The Sidecar resources are used for configuring the sidecar proxies in the service mesh. IstioEgressListener specifies the properties of an outbound traffic listener on the sidecar proxy attached to a workload instance.
+
+In the hosts field, there is the list of hosts exposed to the workload. Each host in the list have the `namespace/dnsName` format where both namespace and dnsName may have non-obvious values. `namespace` may be either `.`, `~`, `*` or an actual namespace name. `dnsName` has to be a FQDN representing a service, virtual service or a service entry. This FQDN may use the wildcard character.
+
+See more information about the syntax of both `namespace` and `dnsName` into https://istio.io/docs/reference/config/networking/sidecar/#IstioEgressListener[istio documentation].
+
+
+==== Resolution
+
+Make sure there is a service, virtual service or service entry matching with the host.
+
+==== Severity
+
+icon:exclamation-triangle[] Warning
+
+==== Example
+
+[source, yaml]
+----
+include::/data/files/validation_examples/904.yaml[]
+----
+
+==== See Also
+
+https://github.com/kiali/kiali/tree/master/business/checkers/sidecars/egress_listener_checker.go[Validator source code, window="_blank"] +
+https://istio.io/docs/reference/config/networking/sidecar/#IstioEgressListener[Sidecar EgressListener documentation, window="_blank"] +
+
+'''
+
+=== KIA1005 - More than one Sidecar applied to the same workload
+
+The Sidecar resources are used for configuring the sidecar proxies in the service mesh. The `workloadSelector` field is the set of labels that selects which workloads' sidecars this resource will be applied to.
+
+The behavior of the system is undefined if two or more Sidecar configurations with a workloadSelector select the same workload instance. It means that Istio undeterministicly applies one sidecar or another to a set of workloads. This scenario is very unlikely and it should be prevented.
+
+==== Resolution
+
+Make sure the labels for the sidecar are applying to a unique set of workloads. Ensure that other sidecars in the namespace are not targetting workloads targetted in this sidecar.
+
+==== Severity
+
+icon:times-circle[] Error
+
+==== Example
+
+[source, yaml]
+----
+include::/data/files/validation_examples/905.yaml[]
+----
+
+==== See Also
+
+https://github.com/kiali/kiali/tree/master/business/checkers/sidecars/multi_match_checker.go[Validator source code, window="_blank"] +
+https://istio.io/docs/reference/config/networking/sidecar[Sidecar documentation: first warning, window="_blank"] +
+
 [#virtualservices]
 == VirtualServices
 
-=== KIA1001 - DestinationWeight on route doesn't have a valid service (host not found)
+=== KIA1101 - DestinationWeight on route doesn't have a valid service (host not found)
```

Comment:
I guess these codes were the last one we discussed, right ?
We can't change the code for errors.

Reply:
Yes, they are the ones we discussed.